crates/turbopack-ecmascript/src/references/esm/url.rs

```diff
@@ -124,21 +124,42 @@ impl ChunkableModuleReference for UrlAssetReference {
 
 #[turbo_tasks::value_impl]
 impl CodeGenerateable for UrlAssetReference {
+    /// rewrites calls to the `new URL()` ctor depends on the current
+    /// conditions. This is based on the configuration (UrlRewriteBehavior),
+    /// and the current context (rendering), lastly the asset's condition
+    /// (if it's referenced / external). The following table shows the
+    /// behavior:
+    /*
+     * original call: `new URL(url, base);`
```

Comment:
Can you clarify if this should be pointing to the output path or the source path for each of these cases?

Reply:
Output path. Webpack points to the output path as well.